tests/redis_stores-test.js

```diff
@@ -7,16 +7,22 @@ const {
 const redis = require('redis');
 const { promisify } = require('util');
 
+const defaultPrefix = 'launchdarkly';
+
 // Runs the standard test suites provided by the SDK's store_tests module.
 
 // This is a single file because if the two test suites were run from separate files, they could
 // be interleaved by Jest. Since our implementation of clearAllData is not very smart and will
 // remove *all* keys with the given prefix, we could step on another test's data if the shared
 // test suites happen to use the same prefix.
 
+function actualPrefix(prefix) {
+  return prefix || defaultPrefix;
+}
```

Comment:
The prefix-related changes here are because the Node SDK shared test suites will now test each database integration both with a prefix specified and with _no_ prefix specified. In the latter case, whatever the default behavior of the data store itself is, the special methods for initializing test data should use that same default behavior.